client/sites-dashboard-v2/site-preview-pane/dotcom-preview-pane.tsx

```diff
@@ -46,78 +48,89 @@ const DotcomPreviewPane = ( {
 	const isSimpleSite = ! site.jetpack;
 	const isPlanExpired = !! site.plan?.expired;
 
-	const features = useMemo(
-		() => [
-			createFeaturePreview(
-				DOTCOM_OVERVIEW,
-				__( 'Overview' ),
-				true,
-				selectedSiteFeature,
-				setSelectedSiteFeature,
-				selectedSiteFeaturePreview
-			),
-			createFeaturePreview(
+	const features: FeaturePreviewInterface[] = useMemo( () => {
+		const isActiveAtomicSite = isAtomicSite && ! isPlanExpired;
+		const siteFeatures = [
+			{
+				id: 'overview',
```

Comment:
I think this is redundant here? We can always use `featureIds[ 0 ]` below as ID, what do you think?

Reply:
I think it's more explicit. But I'm okay with using `featureIds[0]`